CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 0.2.19
+
+### Enhancements
+
+- Indicate when we retried requests during network errors ([#4725](https://github.com/astral-sh/uv/pull/4725))
+
+### CLI
+
+- Add `--disable-pip-version-check` to compatibility arguments ([#4672](https://github.com/astral-sh/uv/pull/4672))
+- Allow `uv pip sync` to clear an environment with opt-in ([#4517](https://github.com/astral-sh/uv/pull/4517))
+- Add `--invert` to `uv pip tree` ([#4621](https://github.com/astral-sh/uv/pull/4621))
+- Omit `(*)` in `uv pip tree` for empty packages ([#4673](https://github.com/astral-sh/uv/pull/4673))
+- Add `--package` to `uv pip tree` ([#4655](https://github.com/astral-sh/uv/pull/4655))
+
+### Bug fixes
+
+- Fix implementation of `GitDatabase::contains` ([#4698](https://github.com/astral-sh/uv/pull/4698))
```

Comment:
Maybe a description of the bug itself rather than the implementation?

Reply:
Yeah want to change that when you add https://github.com/astral-sh/uv/pull/4740?

Reply:
(I presume you're going to sneak that in haha)